sdk/inc/azure/iot/az_iot_hub_client.h

```diff
@@ -41,11 +41,20 @@ enum
  */
 typedef struct
 {
-  az_span module_id; /**< The module name (if a module identity is used). */
-  az_span user_agent; /**< The user-agent is a formatted string that will be used for Azure IoT
-                         usage statistics. */
-  az_span model_id; /**< The model id used to identify the capabilities of a device based on the
-                       Digital Twin document. */
+  /**
+   * The module name (if a module identity is used).
+   */
+  az_span module_id;
+
+  /**
+   * The user-agent is a formatted string that will be used for Azure IoT usage statistics.
+   */
+  az_span user_agent;
+
+  /**
+   * The model id used to identify the capabilities of a device based on the Digital Twin document.

```

Comment:
Since we are updating docs, fix all such instances.
```suggestion
   * The model ID used to identify the capabilities of a device based on the Digital Twin document.
```